modules/lang-mustache/src/test/java/org/elasticsearch/messy/tests/SuggestSearchTests.java

```diff
@@ -0,0 +1,1329 @@
+/*
```

Comment:
this is a full copy of our tests that has been moved back to core into ` org.elasticsearch.search.suggest.SuggestSearchIT` as far as I can tell only `testExactMatch` should be added?